scipy/spatial/transform/rotation.py

```diff
@@ -629,3 +629,31 @@ def inv(self):
         if self._single:
             quat = quat[0]
         return self.__class__(quat, normalized=True)
+
+    def __mul__(self, other):
+        """Returns the composition of the rotations.
```

Comment:
PEP prescribes to write it as "Return the composition of the rotation". But perhaps write it as "Compose this rotation with other."